rendermodules/intensity_correction/apply_multiplicative_correction.py

```diff
@@ -70,50 +72,82 @@ def getImage(ts):
     ts: renderapi.tilespec.TileSpec
         tilespec to get images from
         (presently assumes this is a tiff image whose URL can be read with tifffile)
-
+    channel: str
+        channel name to get image of, default=None which will default to the non channel image pyramid
     Returns
     =======
     numpy.array
         2d numpy array of this image
     """
-    d = ts.to_dict()
```

Comment:
was this d never used?

Reply:
It was an orphan when you added it in 905189524a11e81149f9737faf8702e1005f8d93.